src/ansible/roles/packages/tasks/RedHat8.yml

```diff
@@ -0,0 +1,6 @@
+- name: Enable idm module
+  command: yum module enable idm:DL1 -y
+  when: ansible_distribution_major_version == '8'
```

Comment:
when is not needed here.

Reply:
It is needed after all on rhel 9:
fatal: [master.ipa.test]: FAILED! => changed=true 
  cmd:
  - yum
  - module
  - enable
  - idm:DL1
  - -y
  delta: '0:00:02.163913'
  end: '2023-07-25 06:13:18.094871'
  msg: non-zero return code
  rc: 1
  start: '2023-07-25 06:13:15.930958'
  stderr: |-
    Error: Problems in request:
    missing groups or modules: idm:DL1
  stderr_lines: <omitted>
  stdout: |-
    rhel-9.3 buildroot                               57 MB/s | 3.6 MB     00:00
    rhel-9.3 buildroot debuginfo                     56 MB/s | 2.5 MB     00:00
  stdout_lines: <omitted>